assets/js/common.js

```diff
@@ -506,6 +506,14 @@ var hivepress = {
 							dayElem.innerHTML += '<span class="flatpickr-day-label">' + range.label + '</span>';
 							dayElem.className += ' flatpickr-status';
 
+							if (range.hasOwnProperty('class')) {
```

Comment:
If I understand correctly this part just blocks (greys out) dates with End Date Only label when they are loaded? If so then you can remove it - it's ok to show the End Date Only date as available, it already has a red dot and a tooltip. The code below seems to prevent selecting this date as a start anyway and the user will see the tooltip with the End Date Only label when trying to click it as a start date in any case.

Reply:
P.S. Switch to the "release" branch in Bookings when testing this, I moved the "last-date" change to a new branch.